_raml/apps/coupon-campaign/coupon-campaigns.raml

```diff
@@ -0,0 +1,50 @@
+#%RAML 0.8
+displayName: Coupon campaigns
+post:
+  description: Creates a coupon campaign for a shop. Coupon campaign types can be FIX (absolute value), `PERCENT` (percentage value), and `SHIPPING` (waive delivery costs). *epages6 *epagesNow
+  securedBy: [coupons_write]
+  is: [ locale ]
+  body:
+    application/json:
+      example: !include examples/post_coupon_campaign_request.json
+      schema: !include schema/post_coupon_campaign_request.schema.json
+  responses:
+    201:
+      body:
+        application/json:
+          example: !include examples/post_coupon_campaign_response.json
+          schema: !include schema/post_coupon_campaign_schema.json
+/{couponCampaignId}:
+  get:
+    description: Returns a specific coupon campaign from a shop identified by the provided coupon campaign Id. *epages6 *epagesNow
+    securedBy: [coupons_read]
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include examples/get_coupon_campaign_couponcampaignid_response.json
+            schema: !include schema/get_coupon_campaign_couponcampaignid.schema.json
+  delete:
+    description: Deletes a specific coupon campaign from a shop identified by the provided coupon campaign Id. *epages6 *epagesNow
+    securedBy: [coupons_write]
+    responses:
+      204:
+
+  /coupons:
+    displayName: Coupons
+    put:
+      description: Creates a coupon code in a coupon campaign. *epages6 *epagesNow
+      securedBy: [coupons_write]
+      body:
+        application/x-www-form-urlencoded:
+          formParameters:
+            code:
+              description: The coupon code, e.g. `summer2019`.
+              required: true
+              type: string
+              example: summer2019
+      responses:
+        204:
```

Comment:
Above, you've added 
"body:
          application/json:"
after such a response.. Don't you need to add it here also?